brands/tourism/hotel.json

```
@@ -731,6 +761,26 @@
       "name": "Park Hyatt",
       "tourism": "hotel"
     }
+  },
+    "tourism/hotel|Park Inn": {
+    "matchNames": ["Park Inn"],
+    "tags": {
+      "brand": "Park Inn",
+      "brand:wikidata": "Q60711675",
+      "brand:wikipedia": "en:Park Plaza Hotels & Resorts#Park Inn by Radisson",
```

Russian wiki has a dedicated page for this brand, so we should link there instead of using an internal anchor link to the English.
```suggestion
      "brand:wikipedia": "ru:Park Inn by Radisson",
```